js/plugins/trimTimeline.js

```diff
@@ -52,55 +70,55 @@ var trimClinicalTimeline = function (maxDays, minDays, getZoomLevel, width, getT
         }
       }
       if (i != tickValuesInt.length - 1) {
-        if (ticksToShowKink.indexOf(tickValuesInt[i]) == -1) {
-          ticksToShowKink.push(tickValuesInt[i])
+        if (ticksToShow.indexOf(tickValuesInt[i]) == -1) {
+          ticksToShow.push(tickValuesInt[i])
         }
-        if (ticksToShowKink.indexOf(tickValuesInt[i + 1]) == -1) {
-          ticksToShowKink.push(tickValuesInt[i + 1])
+        if (ticksToShow.indexOf(tickValuesInt[i + 1]) == -1) {
+          ticksToShow.push(tickValuesInt[i + 1])
         }
       }
     }
   });
 
-  ticksToShowKink.sort(function(a, b) { return a - b; })
-
-  if (tickValues.length > 5) {
-      ticksToShowKink.forEach(function (item, index) {
-      //calculate the position to place the kinks
-
-      if (index < ticksToShowKink.length -1) {
-        var first = ticksToShowKink[index];
-        var second = ticksToShowKink[index + 1];
-
-        if (second - first > tolerance) {
-          breakTimelineForKink.push(index + 1)
-        } 
-      }
-    });
-  }
+  ticksToShow.sort(function(a, b) { return a - b; })
+  var prevPush = 0;
+  ticksToShow.forEach(function (item, index) {
+  //calculate the position to place the kinks
```

Comment:
indentation different from next line, try to use eslint/jshint or another code analyzer to prevent this